services/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str, password: str, email: str = None,
+        first_name: str = None, last_name: str = None
+) -> None:
+    user_model = get_user_model()
+    user = user_model.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
```

Comment:
The `create_user` function uses the `create_user` method, which is a good practice as it handles password hashing and other user creation details.